{{cookiecutter.repo_name}}/.pre-commit-config.yaml

```
@@ -1,12 +1,9 @@
 repos:
-  - repo: https://github.com/psf/black
-    rev: 20.8b1
-    hooks:
-      - id: black
-        language_version: python3 # Should be a command that runs python3.6+
-
-  - repo: https://gitlab.com/pycqa/flake8
-    rev: '3.8.4'
-    hooks:
-    - id: flake8
-      args: [--count, --show-source, --statistics]
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.1.4
+  hooks:
+    # Run the linter.
+    - id: ruff
```

I found this config to be quite helpful: automaticall print the issues, or autofix them if possible.

```suggestion
    - id: ruff
          args: [--show-source, --fix]
```